android/settings.gradle

```diff
@@ -7,6 +7,7 @@ pluginManagement {
 
 plugins {
     id("com.android.library").version("8.3.0").apply(false)
+    id("org.jetbrains.kotlin.android").version("1.9.24").apply(false)
```

Comment:
Will adding kotlin to our library cause any integration changes for the client?

Reply:
The kotlin version here doesn't match the one defined by ext.kotlin_version

Reply:
fixed this, removed config from `settings.gradle`

There will be no changes for client integration.